pkg/generated/openapi/openapi_test.go

```diff
@@ -44,6 +45,12 @@ func TestOpenAPIRoundtrip(t *testing.T) {
 				return
 			}
 
+			// Remove the embedded v2 schema if it presents.
```

Comment:
I don't understand why this is part of the PR?

Reply:
This PR reveals this issue.

Reply:
The embedded v2 schema becomes an `map[string]interface{}` after unmarshalling, which fails the DeepEqual